spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/source/SparkPositionDeltaWrite.java

```diff
@@ -185,6 +196,7 @@ public void commit(WriterCommitMessage[] messages) {
 
       int addedDataFilesCount = 0;
       int addedDeleteFilesCount = 0;
+      int removedDeleteFilesCount = 0;
```

Comment:
Good point, it's not being used but I'd rather include it since it will be useful especially as this is a new thing we're adding. I'll update the logs